### ISSUE TITLE
App says it won't run on a rooted device

### ISSUE THREAD

**Issue by majid4466** [anbox/anbox#1986]
I have installed a OTP password generator from my bank on anbox. When I open the app though it shows a message that translates to this:

> To keep your data secure this app does not run on rooted devices.

Is anbox a _rooted device_? Is there a way to hide that from the app?

**Comment by Fuseteam** [comment]
yes anbox is rooted, i heard magisk can hide that but i have no experience with that
